src/test/java/updated/game/abstractClassImpl/TicTacToe.java

```diff
@@ -0,0 +1,158 @@
+package updated.game.abstractClassImpl;
+
+import updated.game.abstractClasses.*;
+
+import java.util.List;
+import java.util.Scanner;
+
+import static java.lang.System.exit;
+import updated.game.abstractClasses.*;
+
+public class TicTacToe extends Game {
+    Human human;
+    Gamer computer;
+    public TicTacToe(){
+        super();
+        human = new Human();
+        computer = new EasyComputer();
+        this.board = new TicTacToeBoard();
+
+    }
+
+    @Override
+    public void printInstructions() {
+        System.out.println("******************************");
```

Comment:
Create a method print() for code readability
Can use forloop

Reply:
Done